galaxy_api/api/ui/viewsets/namespace.py

```diff
@@ -49,9 +51,18 @@ class NamespaceViewSet(
 
     filterset_class = NamespaceFilter
 
+    def create(self, request, *args, **kwargs):
+        serializer = serializers.NamespaceSerializer(data=request.data)
+        if serializer.is_valid(raise_exception=True):
+            namespace_saved = serializer.save()
+        return Response({
```

Comment:
This should be either serialized body of a created object or empty response with HTTP code `201 Created`